packages/main/src/Button.ts

```diff
@@ -347,6 +347,9 @@ class Button extends UI5Element implements IFormElement, IButton {
 	}
 
 	onEnterDOM() {
+		if (isDesktop()) {
+			this.setAttribute("desktop", "");
+		}
 		this._isTouch = (isPhone() || isTablet()) && !isCombi();
```

Comment:
The `_isTouch` private property isn't currently used. If it's not needed we could remove it.